### PR TITLE
reST fixes for contributors files

### PR DESCRIPTION
**Pull request by stevepiercy**

### CODE REVIEW

**Comment by stevepiercy** [comment]
I wondered what that meant in GitHub. I got this from the master branch.
https://github.com/audreyr/cookiecutter/blame/master/Makefile

Strange how it was not caught until I ran it `make contributing` and made this PR. *shrug*

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -1,461 +1,465 @@
...
-
-Look through the GitHub issues for features. Anything tagged with "enhancement"
-and "please-help" is open to whoever wants to implement it.
-
-Please do not combine multiple feature enhancements into a single pull request.
-
-Note: this project is very conservative, so new features that aren't tagged
-with "please-help" might not get into core. We're trying to keep the code base
-small, extensible, and streamlined. Whenever possible, it's best to try and 
-implement feature ideas as separate projects outside of the core codebase.
```
Comment (michaeljoseph) [inline CONTRIBUTING.rst]: It looks like this text was updated directly in `CONTRIBUTING.rst` and not in `docs/types_of_contributions.rst`, could you please fix this?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -1,461 +1,465 @@
...
-    cat docs/contributor_setup.rst >> CONTRIBUTING.rst
-    echo "\r\r" >> CONTRIBUTING.rst
-    cat docs/contributor_guidelines.rst >> CONTRIBUTING.rst
-    echo "\r\r" >> CONTRIBUTING.rst
-    cat docs/contributor_tips.rst >> CONTRIBUTING.rst
-    echo "\r\r" >> CONTRIBUTING.rst
-    cat docs/core_committer_guide.rst >> CONTRIBUTING.rst
-    echo "\r\rAutogenerated from the docs via \`make contributing\`" >> CONTRIBUTING.rst
-    echo "WARNING: Don't forget to replace any :ref: statements with literal names"
-    WARNING: Don't forget to replace any :ref: statements with literal names
```
Comment (michaeljoseph) [inline CONTRIBUTING.rst]: Can we update this in `docs/core_committer_guide.rst` to be consistent with your change in the  `Makefile`?

Alternatively, I'm not opposed to just removing this output entirely.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (stevepiercy) [inline CONTRIBUTING.rst]: @michaeljoseph on my branch, `docs/types_of_contributions.rst` is current with master. I think that someone may forgotten to run `make contributing` and commit `CONTRIBUTING.rst` at some point, so the source files were getting updated but not the concatenation. I nearly missed it myself.

File: CONTRIBUTING.rst (hunk 2 above)
Comment (stevepiercy) [inline CONTRIBUTING.rst]: Fixed in 77d44ad3ecbeae1ed95ceb7127216f2d230ebd63

File: CONTRIBUTING.rst (hunk 1 above)
Comment (michaeljoseph) [inline CONTRIBUTING.rst]: @stevepiercy I think the issue here is that the concatenation got updated, but not the source file.

https://github.com/audreyr/cookiecutter/commit/369553423d80c9054d69bb1b7aff1443571bc7d6#diff-ea1f65a8eaac0cec7ae151dfdbbbc587R56

Comment (stevepiercy) [inline CONTRIBUTING.rst]: @michaeljoseph OK, I double-checked, and I found that both the commit history and the raw content of the file `docs/types_of_contributions.rst` on both `audreyr/master` and `stevepiercy/rest-syntax` are identical.

Compare history:
* https://github.com/audreyr/cookiecutter/commits/master/docs/types_of_contributions.rst
* https://github.com/stevepiercy/cookiecutter/commits/rest-syntax/docs/types_of_contributions.rst

Compare source:
* https://raw.githubusercontent.com/audreyr/cookiecutter/master/docs/types_of_contributions.rst
* https://raw.githubusercontent.com/stevepiercy/cookiecutter/rest-syntax/docs/types_of_contributions.rst

Do you see something that I missed?

In the commit you linked to, it appears that @audreyr on Jun 7, 2016 made the commit to only 1 file, `/CONTRIBUTING.rst`, but not the source file.

Would you like me to take the changes from @audreyr's commit in `/CONTRIBUTING.rst` and put them into `docs/types_of_contributions.rst` on my branch, then render `/CONTRIBUTING.rst` again and commit both files? I think that is what you want in the end, but want to confirm. Thank you!

Comment (michaeljoseph) [inline CONTRIBUTING.rst]: > Would you like me to take the changes from @audreyr's commit in /CONTRIBUTING.rst and put them into docs/types_of_contributions.rst on my branch, then render /CONTRIBUTING.rst again and commit both files?

Yes please :) Thanks for being so much clearer than my request 👍

Comment (stevepiercy) [inline CONTRIBUTING.rst]: Fixed in https://github.com/audreyr/cookiecutter/pull/878/commits/bace55295fadab30a491a51fd49a1ccc09aabed4

**Comment by michaeljoseph** [comment]
Thanks @stevepiercy 🎉

**Comment by hackebrot** [comment]
Woohoo! Thank you @stevepiercy 👏

**Pull request merged**